.github/workflows/jvm-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: JVM CI with Maven
+
+on:
+  push:
+    branches: [ '**' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        scala-version: [2.11, 2.12]
+        spark-version: ["2.4", "3.0"]
```

Comment:
here the version are as string but later (lines 21 and 23) you check them as float, is it correct? can we have them both as float?

Reply:
yes it's the only way that works as expected because if I use `spark-version: [2.4, 3.0]` in case of `3.0` the value got coerced into `3` 🤷‍♂️

Reply:
ok, I'm pretty sure it works for me on the python tests